popgym/envs/minesweeper.py

```diff
@@ -106,9 +105,8 @@ def reset(
                     np.roll(src_grid, shift_i, 0), shift_j, 1
                 )[1:-1, 1:-1]
 
-        random_start = tuple(np.random.randint(self.dims))
         self.timestep = 0
-        obs = np.array([self.neighbor_grid[random_start], *random_start])
+        obs = 0
```

Comment:
Similar to above, will the agent get an incorrect reading for `0` if a mine exists there?